Autotype

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python3
+import os
+import typer
+from typing import Optional
+from Simulator import Type
+import time
+
+def autotype_cli(
```

Comment:
Does this seem to be redundant cant we import `autotype_cli` from `command_line_script.py`?

Reply:
So Basically this file is a Unix-executable-file. So, the thing is, it should be standalone from the `command_line_script.py`. Technically the Right way to structure these executable files was to put them all in a `bin` folder but, Autotype is having some dependency problems.

Reply:
I think I am a bit confused.

I tried making the `command_line_script.py` and `GUI_script.py` executable by adding the shebang `#! /usr/bin/env python3` to both these files and make them executable `chmod +x` 

After exporting the paths in the current shell, I was able to call `command_line_script.py` and `GUI_script.py` directly (from anywhere in my filesystem in that shell) without appending them with `python` (shebang does the job)

So either `Autotype` could take args like `cli and gui` and call the respected functions accordingly.

If its still confusing we can get on a meet and clear this out. 
Let me know your thoughts